frontend/src/components/CreateEvent/CreateEvent.jsx

```diff
@@ -1,10 +1,90 @@
 import "./CreateEvent.css"
+import { useState } from "react";
 
-function CreateEvent () {
+function CreateEvent ({isOpen, onClose, onSubmit}) {
```

Comment:
Think ahead of time how you will create the EditEvent modal as well. Will they share the same component? Will they share similar UI pieces?